.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 extension/dist
 extension.zip
+ext/main.js
```

Comment:
The current version in `master` outputs the bundle in `extension/dist`, so this should be fine as is! Maybe double check your branch is rebased off of latest `master`?